test/Activity-test.js

```diff
@@ -69,4 +69,8 @@ describe('Activity', () => {
     expect(activity.returnTwoDayStairStreak()).to.eql(['2019/06/26', '2019/06/25']);
   });
 
+  it('should test inheritance', () => {
+    expect(activity.returnIndividualStatForDate('2019/06/15', 'numSteps')).to.equal(3577);
+    expect(activity.returnIndividualStatForDate('2019/06/24', 'flightsOfStairs')).to.equal(15);
+  });
```

Comment:
I had initially made a big push to still check each test in each child class to ensure that it could still pass even though it was pulling the method/properties from the parent class now.   After checking in with Brittany, she stated that we wouldn't need to do this because our test suite doesn't need to test that javaScripts's inheritance process works.  We just need to test that our methods work and testing them in the Utility class accomplishes that.   

We may want to remove these based on that feedback.  My apologies for misdirecting us - I seem to always want to be overly thorough in testing which isn't necessary.  I think we can merge this and leave them for now and remove them later if both of you agree.

Reply:
I've built a new issue so that we can go back in later and remove these unnecessary tests. We can include it in a more holistic effort to further bolster clean our testing later on while we focus on other things for now.

With that issue cached, I'm going to merge these file changes.